server/src/main/java/io/spine/server/event/EventBus.java

```diff
@@ -148,7 +154,7 @@ boolean hasDispatchers(EventClass eventClass) {
 
     @Override
     protected DeadMessageTap<EventEnvelope> getDeadMessageHandler() {
```

Comment:
Can we have a more general type returned? Please also note my question about the `handler` vs `tap` names regarding the field.